wake/demo.wake

```diff
@@ -44,18 +44,13 @@ global def demo =
   def programName = "demo"
   def cFiles =
     source "{blockPIOSifiveRoot}/tests/demo/main.c",
-    source "{blockPIOSifiveRoot}/metal/drivers/pio/sifive_pio0.h",
-    source "{blockPIOSifiveRoot}/metal/drivers/sifive_pio.c",
     Nil
-
   makeTestProgramPlan programName cFiles
-  | setTestProgramPlanIncludeDirs ("{blockPIOSifiveRoot}/metal/drivers", Nil)
   | setTestProgramPlanCFlags ("-DPIO_BASE=0x60000", Nil)
```

Comment:
We should remove this line, since we should ensure that we don't depend on having this macro set anymore.